docs/codeql/ql-language-reference/annotations.rst

```diff
@@ -433,7 +433,6 @@ The ``bindingset`` annotation takes a comma-separated list of variables.
   (for characteristic predicates and member predicates) and ``result`` (for predicates that return a result). 
   For more information, see ":ref:`predicate-binding`."
 - When you annotate a class, each variable must be ``this`` or a field in the class. 
-  Binding sets for classes are supported from release 2.3.0 of the CodeQL CLI, and release 1.26 of LGTM Enterprise.
```

Comment:
I'm not sure what the policy is do we want to completely remove this, or do we still want to keep the part that refers to the CLI (similar for some other removals below)?

Reply:
Thanks for raising this. I looked back to see when we released 2.3.0 of the CLI and thought that since it was released over 2 years ago, it was reasonable to assume that this note was no longer needed: https://github.com/github/codeql-cli-binaries/releases/tag/v2.3.0.

Reply:
Thanks for the clarification.